Source/gmenu.cpp

```diff
@@ -342,10 +342,10 @@ BOOL gmenu_left_mouse(BOOL isDown)
 	if (MouseY >= PANEL_TOP) {
 		return FALSE;
 	}
-	if (MouseY - 117 < 0) {
+	if (MouseY - (115 + UI_OFFSET_Y) < 0) {

```

Comment:
Why the need to change it from 117 to 115?

Reply:
This was the pause menu, right? I remember mouse navigation not working right until I changed something there. If it works fine with 117, then let's just use that.